src/main/java/org/prebid/server/validation/RequestValidator.java

```diff
@@ -365,17 +374,72 @@ private static Request parseNativeRequest(String rawStringNativeRequest, int imp
         }
     }
 
-    private void validateNativeContext(Integer context, int index) throws ValidationException {
-        if (context != null && (context < 1 || context > 3)) {
-            throw new ValidationException(
-                    "request.imp[%d].native.request.context must be in the range [1, 3]. Got %d", index, context);
+    private void validateNativeContextTypes(Integer context, Integer contextSubType, int index)
+            throws ValidationException {
+
+
+        if (context != null) {
```

Comment:
I'm not sure that this `if` is needed, type could be initialized as follows `final int type = context != null ? context.intValue() : 0`. Same is true for `subType`